JUnit version error

Hi,
When trying to execute
`mvnw clean package` I get the following error:

> [ERROR] Failed to execute goal on project clicklogger: Could not resolve dependencies for project com.clicklogs:clicklogger:jar:1.0-SNAPSHOT: Could not find artifact junit:junit:jar:5.0-SNAPSHOT, try downloading from https://github.com/junit-team/junit/wiki/Download-and-Install -> [Help 1]
> org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project clicklogger: Could not resolve dependencies for project com.clicklogs:clicklogger:jar:1.0-SNAPSHOT: Could not find artifact junit:junit:jar:5.0-SNAPSHOT, try downloading from https://github.com/junit-team/junit/wiki/Download-and-Install

--- a/source/clicklogger/pom.xml
+++ b/source/clicklogger/pom.xml
@@ -92,7 +92,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>[4.13.1,)</version>
+      <version>4.13.1</version>
     </dependency>
   </dependencies>
 

